File: client/src/models/Users/UserListDto.ts
import type { CompanyUserListDto } from '@/models/CompanyUsers/CompanyUserListDto'

export class UserListDto{
    id: string = "";
    userName: string = "";
    email: string = "";
    phoneNumber: string = "";
    firstName: string = "";
    lastName: string = "";
    fullName: string = "";
    isDeleted: boolean = false;
    companyUsers: CompanyUserListDto[] = [];
    isAdmin: boolean = false;
}

File: server/EAccountingServer.Application/Models/Dtos/Users/UserListDto.cs
using EAccountingServer.Domain.Entities;

namespace EAccountingServer.Application.Models.Dtos.Users
{
    public class UserListDto
    {
        public Guid Id { get; set; }
        public string UserName { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string FullName { get; set; }
        public ICollection<CompanyUser> CompanyUsers { get; set; }
        public bool IsDeleted { get; set; }
        public bool IsAdmin { get; set; }
    }
}
